CHANGELOG.md

```diff
@@ -32,6 +33,45 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.28.1 - 2023-03-01
+
+### Release notes
+
+### Added
+
+- Enable rook-ceph network polices by default for exoscale
+- Troubleshooting scripts for HNC
+- Tests for Groups RBAC
+- the possibility to add static users for opensearch
+- Support for extracting and storing audit logs with Fluentd
+- Compaction for logs stored directly in object store by Fluentd
+- New 'Log review overview' Opensearch dashboard
+- Added falco rules to ignore redis operator related alerts.
+
+### Fixed
+
+- The update-ips script can now fetch Calico Wireguard IPs
+- The update-ips script can now fetch object storage sync destination IPs
+- The test scripts can now always access the kubeconfig
+- The OpenSearch network policies now properly work with dedicated nodes and shapshots enabled
+- The `clean-sc` script now patches any pending challenges which would prevent the removal of certain namespaces
+
+### Updated
+
+- The NetworkPolicy Dashboard have been updated to be more clear
+- Upgraded the Starboard-operator helm chart to `0.10.11` which upgrades the app version to `0.15.11`
+
+### Changed
+
+- Thanos Rulers now sends alerts to all Alertmanagers in an HA setup, sends requests directly to Queries, and are now accessible by Queries
+- Thanos components now use DNS SD to automatically find healthy replicas
+- Alerts can now be inspected in ops Grafana
+- Updated metrics-server chart to 0.6.2 which upgrades metrics-server to 3.8.3
+- Allow the creation of arbitrary network-policy in wc
+- Network polices
+  - Added missing network policy for rook-ceph-csi-detect-version for rook-ceph v1.10
```

Comment:
Something must've gone wrong as all of these things didn't change in this patch.

Reply:
Fixed
Please recheck
Thx